<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BolusScreen</class>
 <widget class="QWidget" name="BolusScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <!-- Tandem logo button -->
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Tandem LOGO</string>
   </property>
  </widget>
  <!-- Back button -->
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <!-- Confirm button -->
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>âœ“</string>
   </property>
  </widget>
  <!-- New INSULIN button -->
  <widget class="QPushButton" name="insulinButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>INSULIN</string>
   </property>
  </widget>
  <!-- Carbs Button -->
  <widget class="QPushButton" name="carbsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0
grams</string>
   </property>
  </widget>
  <!-- BG Button -->
  <widget class="QPushButton" name="bgButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add BG</string>
   </property>
  </widget>
  <!-- View Calculation button -->
  <widget class="QPushButton" name="viewCalcButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>View Calculation</string>
   </property>
  </widget>
  <!-- Label for CARBS -->
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CARBS</string>
   </property>
  </widget>
  <!-- Label for GLUCOSE -->
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GLUCOSE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
